cmd/kube-controller-manager/app/options/options.go

```diff
@@ -113,6 +114,7 @@ func NewCMServer() *CMServer {
 		NodeMonitorPeriod:                 5 * time.Second,
 		ClusterName:                       "kubernetes",
 		TerminatedPodGCThreshold:          12500,
+		EventTTL:                          1 * time.Hour,
```

Comment:
GIven that we have a separate etcd now for events, should we bumb up this period to a week? 1 hour was never that useful for users.
cc @saad-ali